### ISSUE TITLE
Not hashing passwords 

### ISSUE THREAD

**Issue by qarth** [thewhitetulip/web-dev-golang-anti-textbook#53]
You are teaching people some bad practices by not hashing passwords. You should never store plain text passwords.

**Comment by thewhitetulip** [comment]
Yes, I have specifically mentioned that you have to hash passwords and because this is a toy app we are not hashing the password.
